tests/integration/test_replay_event.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+from requests.exceptions import HTTPError
+from sentry_sdk.envelope import Envelope
+
+
+def generate_replay_event():
```

Comment:
TODO: add a test for sending a replay event and replay recording in the same payload